distribute_setup.py

```diff
@@ -452,13 +452,7 @@ def _extractall(self, path=".", members=None):
         self.extract(tarinfo, path)
```

Comment:
I'm inclined to entirely drop this file; we're not going to use it going forward.

Reply:
The whole file? Love to!

Reply:
@offbyone Gone.